src/layouts/EditPage.jsx

```diff
@@ -219,9 +219,11 @@ const EditPage = ({ match, history }) => {
           if (isXSSViolation) setShowXSSWarning(true)
           else {
             updatePageHandler({
-              frontMatter: pageData.content.frontMatter,
-              sha: currSha,
-              pageBody: editorValue,
+              pageData: {
```

Comment:
just double checking - this is required because the hook passes the data to `extractPageInfo`, which takes `{ data, pageData, resourceRoomName }` right

Reply:
Yep!